### DISCUSSION TITLE
Pause and resume while audio and video capturing

### DISCUSSION THREAD

**Discussion by 8JupiterMoll8** [raspberrypi/picamera2#701] [General]
Hi.
Does Picamera2 support pause and resume while audio and video capturing?
Thx.

**Comment by davidplowman** [comment]
Whilst you could stop feeding video frames to the video encoder, I don't think there's any similar control over audio. If both audio and video are required, you might want to consider saving separate files and joining them together afterwards (using a tool like FFmpeg, which could do this extremely quickly). Does that help?

**Comment by 8JupiterMoll8** [comment]
THX. 
I will try this solution . An report my results.
